create.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+#set -o xtrace
+# 
+# Patch EKS nodes
+# 
+
+if [ -z $1 ]; then
+    echo "Error: cluster variable is not set, I don't know which cluster you want to create"
+    exit 1
+fi
+
+export WORKSPACE=$1
+
+# build network and EKS masters
+pushd infra
+rm -rf .terraform
+terraform init -backend-config ../workspaces/$WORKSPACE/backend.cfg 
+terraform plan -input=false -var-file="../workspaces/$WORKSPACE/terraform.tfvars" 
```

Comment:
Seems unnecessary - what does plan do for you that apply will not do again?

Reply:
Intent is to provide some rudimentary linting before trying to build the infrastructure, but given the time taken for a deploy I'll remove it. (tflint is better anyway)

Reply:
Almost said that might be the case but didn't want to presume.
Agree better to just one shot it, it apply will produce the same error if there is one